plugin/storage/es/esCleaner.py

```diff
@@ -60,13 +64,36 @@ def main():
 def filter_main_indices(ilo, prefix):
     ilo.filter_by_regex(kind='prefix', value=prefix + "jaeger")
     empty_list(ilo, "No indices to delete")
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-span-read'], exclude=True)
```

Comment:
Why are these included, aren't they related to rollover?

Reply:
They are excluded, see the last parameter

Reply:
Sorry poor use of words, I mean why are these lines included/added - as they seem to relate to aliases related to the rollover approach. From looking at the tests, it seems like it is so that existing aliases (from rollover) are not touched - but that would imply the user has gone from a rollover environment back to using daily indices?

Reply:
Yes, I have simplified this in the last commit. Now there is a regex which matches exactly the daily inidices so no need to remove rollover indices afterwards.